experiment/process-e2e-suite-log/fileSplitter.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+type FileElement struct {
```

Comment:
What other packages use FileElement? If none please consider making this local to the package: `type fileElement struct{}`

Same for other structs (do not automatically capitalize structs, `foo` is package private whereas `Foo` is public, kind of similar to python's `foo` vs `_foo` except enforced by the compiler

Reply:
Thanks for pointing this out, done :)